src/app/forms/templates/psv/psv-annual.template.ts

```diff
@@ -291,7 +284,64 @@ export const PsvAnnual: FormNode = {
               name: 'particulateTrapSerialNumber',
               label: 'Particulate trap serial number',
               type: FormNodeTypes.CONTROL
+            },
+            {
+              name: 'visitSection',
+              label: 'Visit',
+              type: FormNodeTypes.SECTION
+            },
+            {
+              name: 'testFacilityCombination',
+              label: 'Test facility name/number',
+              type: FormNodeTypes.COMBINATION,
+              options: {
+                leftComponentName: 'testStationName',
+                rightComponentName: 'testStationPNumber',
+                separator: ' / '
+              }
+            },
+            {
+              name: 'testStationName',
+              label: 'Test Station Name',
+              value: '',
+        
```

Comment:
nitpick - we don't need the empty `value: '',` or the blank line here (and at 314)